packages/lit-dev-content/site/tutorials/index.html

```diff
@@ -0,0 +1,149 @@
+---
+title: Tutorials
+---
+
+{% extends 'default.html' %}
+
+{% block head %}
+  {% inlinecss "tutorial.css" %}
+  <script type="module" src="{{ site.baseurl }}/js/components/litdev-tutorial-card.js"></script>
+  <style>
```

Comment:
Nit: Since we inlinecss from the css file, is it worth putting this CSS in a file?